src/test/crux-messenger/secure-cruxid-messenger-prod-test.ts

```diff
@@ -0,0 +1,62 @@
+import * as chai from "chai";
+import sinon from "sinon";
+import chaiAsPromised from "chai-as-promised";
+import 'mocha';
+import {SecureCruxIdMessenger} from "../../core/domain-services";
+import {BasicKeyManager, CruxNetPubSubClientFactory} from "../../infrastructure/implementations";
+import {patchMissingDependencies, getCruxdevCruxDomain} from "../test-utils";
+import {getCruxUserRepository} from "../../application/clients";
+import {CruxSpec} from "../../core/entities";
+
+patchMissingDependencies();
+
+chai.use(chaiAsPromised);
+chai.should();
+const expect = require('chai').expect;
+
+
+const user1PvtKey = "KyBuSe1MMV6NjJgZfyWuvgmxUeAYswLC2HrfYLUri9aP3AS5FBfr"; // release020@cruxdev.crux
+const user2PvtKey = "cdf2d276caf0c9c34258ed6ebd0e60e0e8b3d9a7b8a9a717f2e19ed9b37f7c6f"; // mascott6699@cruxdex.crux
+
+describe('Test Secure Crux Messenger - PROD', function() {
+    beforeEach(async function() {
+        const HOST = "127.0.0.1";
```

Comment:
next task is to make server gracefully start up and stop automatiacally in test cases that require it

Reply:
yes, will figure out and discuss ways to do it